data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/rac_templates.lst

```diff
@@ -52,4 +52,7 @@ Yuan-Ti (Tainted One)
 #Elf ~ Moon																	VISIBLE:NO																																																																																																																																																			FAVOREDCLASS:Wizard																																LANGBONUS:Auran,Chondathan,Gnoll,Gnome,Halfling,Illuskan,Sylvan						SUBRACE:YES
 
 
-Reincarnated into Mountain Orc	KEY:Reincarnated ~ Mountain Orc	VISIBLE:EXPORT	TYPE:Reincarnation	SIZE:M	BONUS:STAT|STR|4|TYPE=Racial	BONUS:VAR|DarkvisionRange|60|TYPE=Base	ABILITY:Special Ability|AUTOMATIC|Orc ~ Light Sensitivity|Orc ~ Orc Blood|Remove Race Traits	MOVE:Walk,30
+# Orc Blood doesnt exist,only Half-Orc blood !
+#Orc ~ Orc Blood
+Reincarnated into Mountain Orc	KEY:Reincarnated ~ Mountain Orc	VISIBLE:EXPORT	TYPE:Reincarnation	SIZE:M	BONUS:STAT|STR|4|TYPE=Racial	BONUS:VAR|DarkvisionRange|60|TYPE=Base	ABILITY:Special Ability|AUTOMATIC|Orc ~ Light Sensitivity| |Remove Race Traits	MOVE:Walk,30
```

Comment:
This should be throwing an error.   |Orc ~ Light Sensitivity| |Remove Race Traits. There should not be a '| |' in there.